.travis.yml

```diff
@@ -5,6 +5,12 @@ addons:
     paths:
       - artifacts/
 
+# Build on pushes only to given branches
+branches:
+  only:
+    - master
+    - /3\.\d+\.x/
```

Comment:
I think explicit would be better, just list out `3.10.x` and `3.12.x`. It is much more clear, even if we have to update it once every 18 months.